zebedee-cms/src/main/java/com/github/onsdigital/zebedee/permissions/cmd/CMDPermissionsService.java

```diff
@@ -38,7 +38,7 @@
      * service account is valid but does not have any permissions.
      * @throws PermissionsException problem getting the service permissions.
      */
-    CRUD getServiceDatasetPermissions(GetPermissionsRequest getPermissionsRequest) throws PermissionsException;
+    CRUD getServiceDatasetPermissions(GetPermissionsRequest request) throws PermissionsException;
```

Comment:
Might be worht updating annotations, `@param getPermissionsRequest {@link GetPermissionsRequest} specifies the request details` -> `@param request {@link GetPermissionsRequest} specifies the request details`